tests/test_recorders.py

```diff
@@ -1882,6 +1882,16 @@ class TestEventRecorder:
     """ Tests for EventRecorder """
     funcs = {"min": np.min, "max": np.max, "mean": np.mean, "median": np.median, "sum": np.sum}
 
+    def test_load(self, cyclical_storage_model):
+        """Test load method""" 
+        m = cyclical_storage_model
+        strg = m.nodes['Storage']
+        param = StorageThresholdRecorder(m, strg, 4.0, predicate='<=', name="trigger")
+        EventRecorder.load(m, {"name": "event_rec", "threshold": "trigger", "tracked_parameter":"inpt_flow"})
+        EventDurationRecorder.load(m, {"event_recorder": "event_rec"})
+        EventStatisticRecorder.load(m, {"event_recorder": "event_rec"})
+        m.step()
```

Comment:
Change this to `m.run()`. I have a feeling something bad happens if you don't specify `recorder_agg_func` or `agg_func`, but it won't happen until the `finish` method.